js/skin.js

```diff
@@ -432,6 +434,22 @@ var Skin = React.createClass({
           </ContentScreen>
           );
           break;
+        case CONSTANTS.SCREEN.MULTI_AUDIO_SCREEN:
+          screen = (
+            <ContentScreen
+              {...this.props}
+              cssClass="oo-close-button oo-close-button--ma"
```

Comment:
Should there be two dashes ```--``` before the ```ma```?

Reply:
Yeah I think it's a part BEM (modifier).